Jenkinsfile.pr-full-test

```diff
@@ -0,0 +1,110 @@
+                    deleteDir()
+                }
+            }
+        }
+        stage('Initialize') {
+            steps {
+                sh "mkdir -p ${WORKING_DIR} && cd ${WORKSPACE} && cp -Rap * ${WORKING_DIR}"
+                sh "set +x && oc login --token=\$(oc whoami -t) --server=${OPENSHIFT_API} --insecure-skip-tls-verify"
+            }
+        }
+        stage('Check if Koigto Operator image exists') {
+            steps {
+                dir ("${WORKING_DIR}") {
+                    script {
+                        operatorImageNotExistsInOpenshift = sh (
+                            script: "set +x && oc get istag -n openshift kogito-cloud-operator:pr-\$(echo \${GIT_COMMIT} | cut -c1-7)",
+                            returnStatus: true
+                        ) != 0
+
+                        if (operatorImageNotExistsInOpenshift) {
+                            echo "Kogito Operator image not exists. Image will be build in following step."
+                        } else {
+                            echo "Kogito Operator image exists. Skippng image build step. "
+                        }
+                    }
+                }
+            }
+        }
+        stage('Build Kogito Operator') {
+            when {
+                expression {
+                    return operatorImageNotExistsInOpenshift
+                }
+            }
+            steps {
+                dir ("${WORKING_DIR}") {
+                    sh """
+                        export GOROOT=`go env GOROOT`
+                        GO111MODULE=on
+                        go get -u golang.org/x/lint/golint
+                        touch /etc/sub{u,g}id
+                        usermod --add-subuids 10000-75535 \$(whoami)
+                        usermod --add-subgids 10000-75535 \$(whoami)
+                        cat /etc/subuid
+                        cat /etc/subgid
+                        make image_builder=podman
+                    """
+                }
+            }
+        }
+        stage('Build Kogito CLI') {
+            steps {
+                dir ("${WORKING_DIR}") {
+                    sh "make build-cli"
+                }
+            }
+        }
+        stage('Push Operator Image to Openshift Registry') {
+            when {
+                expression {
+                    return operatorImageNotExistsInOpenshift
+                }
+            }
+            steps {
+                dir ("${WORKING_DIR}") {
+                    sh """
+                        set +x && podman login -u jenkins -p \$(oc whoami -t) --tls-verify=false ${OPENSHIFT_REGISTRY}
+                        cd version/ && TAG_OPERATOR=\$(grep -m 1 'Version =' version.go) && TAG_OPERATOR=\$(echo \${TAG_OPERATOR#*=} | tr -d '"')
+                        podman tag quay.io/kiegroup/kogito-cloud-operator:\${TAG_OPERATOR} ${OPENSHIFT_REGISTRY}/openshift/kogito-cloud-operator:pr-\$(echo \${GIT_COMMIT} | cut -c1-7)
```

Comment:
could we put this in environment variable ?
like
`PR_NUMBER=${sh(script:"echo \${TAG_OPERATOR#*=} | tr -d '\"'", returnStdOut: true)}`

or in `Initialize` stage:
`env.PR_NUMBER=sh(script:"echo \${TAG_OPERATOR#*=} | tr -d '\"'", returnStdOut: true)`

Reply:
could also be changed in other Jenkinsfiles ;)

Reply:
+1

Reply:
yes, good idea. I'll open a new ticket for this topic

Reply:
https://issues.redhat.com/browse/KOGITO-1679